src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.CommandInvalidException;
+
+import java.io.FileNotFoundException;
+
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+    private static final String CURRENT_WORKING_DIRECTORY = System.getProperty("user.dir");
+    private static final String PATH = "/list.txt";
+
+
+    /**
+     * Constructor for Duke
+     *
```

Comment:
Perhaps consider only having 1 line between your method descriptor and the @param line. There are also other places in your code where you have extra lines within your javadoc code. You can consider removing them for neater code ☺️

Reply:
Did not think about it before, thanks for the suggestion.